src/test/java/org/ow2/proactive/catalog/service/WorkflowXmlManipulatorTest.java

```diff
@@ -82,31 +84,106 @@
+                                                                                   "    onTaskError=\"continueJobExecution\"\n" +
+                                                                                   "     maxNumberOfExecution=\"2\"\n" +
+                                                                                   ">\n" + "    <taskFlow>\n" +
+                                                                                   "    <task name=\"Task1\">\n" +
+                                                                                   "      <genericInformation>\n" +
+                                                                                   "         <info name=\"insideTaskGenInfo\" value=\"TaskGenInfoValue\"/>\n" +
+                                                                                   "       </genericInformation>\n" +
+                                                                                   "      <scriptExecutable>\n" +
+                                                                                   "        <script>\n" +
+                                                                                   "          <code language=\"javascript\">\n" +
+                                                                                   "            <![CDATA[\n" +
+                                                                                   "print(java.lang.System.getProperty('pas.task.name'))\n" +
+                                                                                   "]]>\n" + "          </code>\n" +
+                                                                                   "        </script>\n" +
+                                                                                   "      </scriptExecutable>\n" +
+                                                                                   "    </task>\n" + "  </taskFlow>\n" +
+                                                                                   "     <description>\n" +
+                                                                                   "        <![CDATA[ Perform anomaly detection of an input image (only pedestrians are supposed to be present) ]]>\n" +
+                                                                                   "     </description>\n" +
+                                                                                   "    <genericInformation>" +
+                                                                                   "      <info name=\"first\" value=\"value1\"/>\n" +
+                                                                                   "      <info name=\"second\" value=\"value2\"/>\n" +
+                                                                                   "    </genericInformation>\n" +
+                                                                                   "</job>").getBytes();
+
```

Comment:
Just a question: isn't it better way to embed this data as external (assest) files?